2024-12-23 04:00:16,873 - ERROR - Authentication request failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f629b65fd30>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
2024-12-23 04:00:16,873 - ERROR - ETL Process for Workly failed: Authentication failed: HTTPSConnectionPool(host='api.workly.uz', port=443): Max retries exceeded with url: /v1/oauth/token (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f629b65fd30>: Failed to resolve 'api.workly.uz' ([Errno -2] Name or service not known)"))
